Samples/HelloWorld/HelloWorld/HelloWorld.csproj

```diff
@@ -12,12 +12,14 @@
     <AssemblyName>HelloWorld</AssemblyName>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
+    <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\</SolutionDir>
+    <RestorePackages>true</RestorePackages>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>$(OrleansSDK)\LocalSilo\</OutputPath>
+    <OutputPath>bin\$(Configuration)\</OutputPath>
```

Comment:
1) I think this is a big enough change in the way we run Samples that we want to discuss it before making any changes to it.
2) This change by itself might be a good idea, or not. I am concerned that now we can't run samples against the latest non official build, but only against the officially published nugets. 
3) We are still thinking about installing SDK as the main way to use Orleans, and nugets are there just to support those who want to consumer from nugets.
4) Not sure the whole change is worth our time and efforts, given so many other higher priority tasks.